src/Microsoft.ML.FastTree/TreeEnsemble/TreeEnsemble.cs

```diff
@@ -16,7 +16,37 @@
 
 namespace Microsoft.ML.Trainers.FastTree.Internal
 {
-    public class TreeEnsemble
+    public class TreeEnsembleModel
```

Comment:
>TreeEnsembleModel [](start = 17, length = 17)

Would `TreeRegressorCollection` be a better name?

Reply:
Sure. I will rename TreeEnsembleModel to TreeEnsemble and TreeRegressor to RegressionTree in iteration 21.

---
In reply to: [252498510](https://github.com/dotnet/machinelearning/pull/2243#discussion_r252498510) [](ancestors = 252498510)